.codeclimate.yml

```diff
@@ -7,21 +49,22 @@ engines:
     enabled: true
   duplication:
     enabled: true
-    exclude_paths:
+    exclude_patterns:
       - spec/**/*
   flog:
     enabled: true
-    exclude_paths:
+    exclude_patterns:
       - db/**/*
       - config/**/*
       - spec/**/*
   rubocop:
     enabled: true
-    exclude_paths:
-      - db/**/*
     channel: rubocop-0-49
-ratings:
-  paths:
-  - app/**
-  - lib/**
-  - "**.rb"
+    config:
+      file: '.rubocop.yml'
+    exclude_patterns:
+      - db/**/*
+exclude_patterns:
```

Comment:
does this exclude files for every check? If that's correct, why are we also excluding these same patterns on specific checks?

Reply:
I'll check it. thanks!